playground/miden-wasm/src/lib.rs

```diff
@@ -328,9 +334,9 @@ fn test_verify_program() {
         "trace_len": 1024
     }"#;
 
-    let proof = expected_test_proof::EXPECTED_PROOF_BYTES.to_vec();
+    let proof = prove_program(asm, input_str).unwrap();
 
-    let result = verify_program(asm, input_str, output_str, proof);
+    let result = verify_program(asm, input_str, output_str, proof.proof.unwrap()).unwrap();
```

Comment:
nit: `proof.proof.unwrap()` is a bit strange. Maybe `proof` above should be renamed to something else (`output`, `result`, etc)?